### PR TITLE
Test (reg/2nd-factor): registration and second factor automation

### PR DESCRIPTION
**Pull request by cerebrl**

### CODE REVIEW

File: tests/e2e/suites/registration.test.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { setupAndGo } from '../utilities/setup-and-go';
+
+describe('Test Basic login flow', () => {
```
Comment (spetrov) [inline tests/e2e/suites/registration.test.ts:3]: Test basic registration flow...

File: tests/e2e/suites/registration.test.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  ['chromium', 'webkit', 'firefox'].forEach((browserType) => {
+    it(`should login successfully and then log out with ${browserType}`, async (done) => {
```
Comment (spetrov) [inline tests/e2e/suites/registration.test.ts:5]: ... similar to the above... "should register successfully..."

File: tests/e2e/suites/registration.test.ts (hunk 1 above)
Comment (cerebrl) [reply tests/e2e/suites/registration.test.ts]: Fixed.

File: tests/e2e/suites/registration.test.ts (hunk 2 above)
Comment (cerebrl) [reply tests/e2e/suites/registration.test.ts]: Fixed.

File: tests/e2e/suites/authn-second-factor.test.ts [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import { setupAndGo } from '../utilities/setup-and-go';
+
+describe('Test Second Factor login flow', () => {
```
Comment (cerebrl) [inline tests/e2e/suites/authn-second-factor.test.ts:3]: Added a test suite for second-factor.

**Pull request merged**